pkg/distributor/distributor.go

```diff
@@ -527,7 +524,7 @@ func (d *Distributor) Push(ctx context.Context, req *logproto.PushRequest) (*log
 				}
 
 				n++
-				validatedLineSize += len(entry.Line)
+				validatedLineSize += util.EntryTotalSize(&entry)
```

Comment:
We should not consider the size of `detected_level` because it is not counted in ingestion volume. The problem with considering the size of `detected_value` is that the volume of dropped data will be greater than the actual ingested data.

Reply:
resolved in the commit https://github.com/grafana/loki/pull/14571/commits/daf77c8cdc3ea5e482dcd5a38a316ab2b3e86619